### ISSUE TITLE
clients

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/js-ng#26]
### Urgent:

- [x] sshkey
- [ ] sshagent
- [x] sshclient (that uses remote executor)
- [x] currency_layer
- [x] sonic ([sonic-client](https://github.com/xmonader/python-sonic-client)) @Omarabdul3ziz 
- [x] redis (forwarding to underlyin client)? should be more explicit?
- [x] git
- [x] github @omarelsheekh
- [x] gdrive @kirolosessam 
- [ ] digital_ocean @abom 
- [x] gedis

### Nice to have
- [x] sendgrid
- [x] twilio
- [x] s3
- [x] trello
- [ ] btc_alpha
- [ ] google_compute
- [ ] packet.net
- [ ] grafana
- [ ] influx
- [ ] itsyou.online
- [ ] etcd
- [ ] syncthing

**Comment by abom** [comment]
Blocked by #16.

**Comment by abom** [comment]
Too old, was opened for the first release (before sdk..etc)

**Comment by abom** [comment]
Too old, was opened for the first release (before sdk..etc)
